src/mumble/AudioOutput.cpp

```diff
@@ -437,6 +437,21 @@ void AudioOutput::initializeMixer(const unsigned int *chanmasks, bool forceheadp
 	}
 }
 
+void AudioOutput::prepareOutputBuffers(unsigned int frameCount, QList< AudioOutputBuffer * > &qlMix,
+									   QList< AudioOutputBuffer * > &qlDel) {
+	// Get the users that are currently talking (and are thus serving as an audio source)
+	auto it = qmOutputs.constBegin();
+	for (int i = 0; i < qmOutputs.count(); i++) {
```

Comment:
Why not use a ranged for-loop?

Reply:
I'm assuming because the previous code was like this, but we should indeed switch to a ranged for-loop.